### PR TITLE
zlib: fix incorrect sha1

### PR DESCRIPTION
**Pull request by bosr**

### CODE REVIEW

File: Library/Formula/zlib.rb [hunk 1]
```diff
@@ -41,5 +41,5 @@ def install
    *gcc*) gcc=1 ;;
 +  *clang*) gcc=1 ;;
  esac
- 
+
```
Comment (dunn) [inline Library/Formula/zlib.rb]: Make sure to keep whitespace in the inline patches.

**Comment by sjackman** [comment]
I see the previous SHA1 of `a4d316c404ff54ca545ea71a27af7dbc29817088`
```
❯❯❯ curl -L http://zlib.net/zlib-1.2.8.tar.gz |sha1sum
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  557k  100  557k    0     0   386k      0  0:00:01  0:00:01 --:--:--  386k
a4d316c404ff54ca545ea71a27af7dbc29817088  -
❯❯❯ curl -L https://downloads.sourceforge.net/project/libpng/zlib/1.2.8/zlib-1.2.8.tar.gz |sha1sum
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0   345    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  557k  100  557k    0     0  24094      0  0:00:23  0:00:23 --:--:--  119k
a4d316c404ff54ca545ea71a27af7dbc29817088  -
```

**Comment by sjackman** [comment]
The SHA1 that you are seeing is of the uncompressed TAR file.
See https://github.com/Homebrew/homebrew-dupes/pull/197 and https://github.com/Homebrew/homebrew-php/issues/1102

**Comment by bosr** [comment]
Yes, you must be right. Sorry about that. I guess that's due to our the weird proxy we have at work. I'm surprised though: this never happened with other packages. 

Thanks for your input. 
Regards,

Romain

**Pull request closed without merging**